### ISSUE TITLE
modal behaviour changed

### ISSUE THREAD

**Issue by mfchung** [angular-ui/bootstrap#4672]
there is a change in the modal behaviour. It can be seen at the modal example on https://angular-ui.github.io/bootstrap/

basically when animation is turned off, it appears the screen behind the modal is very obviously deleted and redrawn as the modal is closed.

**Comment by Foxandxss** [comment]
Hello,

I am not sure what you mean? Could you rephrase your issue or put some screenshot?

**Comment by mfchung** [comment]
Hi,

I mean the Modal dialog window. if you look at the example on  https://angular-ui.github.io/bootstrap/ page in the section Modal (ui.bootstrap.modal) 

click on large modal button and then do it again with the animation button false.

You will see that when the modal window is closed, when the animation is false, the view underneath is set to black  and then redrawn.

**Comment by Foxandxss** [comment]
Ah, I see that. Will investigate.

**Comment by wesleycho** [comment]
What is happening is that the modal backdrop has a background color of `#000` - when the opacity is 1, it will appear to be solid black...should give a good base to investigate though.

**Comment by wesleycho** [comment]
Ah, so the `fade` class is not applied when animations are disabled, so removing the `in` class will cause this effect to happen.
